hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java

```diff
@@ -68,9 +68,25 @@
   // default heap allocator, it will just allocate ByteBuffers from heap but wrapped by an ByteBuff.
   public static final ByteBuffAllocator HEAP = ByteBuffAllocator.createOnHeap();
 
-  public static final String MAX_BUFFER_COUNT_KEY = "hbase.ipc.server.allocator.max.buffer.count";
+  public static final String MAX_BUFFER_COUNT_KEY = "hbase.server.allocator.max.buffer.count";
```

Comment:
Should we keep ``ipc``? If it is for ipc only. Otherwise, it's ok to remove if it can be widely used,

Reply:
Previously the BBPool was used by the IPC/RPC server component alone. The name came that way.  HBase IPC server config. Now it is used other places too.  So better just say the BBPool related configs used in server. BTW we name it like Allocator now instead of direct refer to pool. So fine.

Reply:
Not only,  the allocator will also allocate ByteBuffers for HDFS block reading. So remove the ipc is better now.

Reply:
Good to know, no question about this name.